README.md

```diff
@@ -1,23 +1,24 @@
 # Vita3K Android
 
 This repository contains the public releases and CI (automated build) for the Vita3K Android version.
-Vita3K Android is an experimental PlayStation Vita emulator for Android. It comes as an APK.
+Vita3K Android is an experimental PlayStation Vita emulator for Android. It comes as an APK, ready for you to install.
 
-Minimum requirements are as followed:
-- ARM64 device with
+Minimum requirements are as follow:
+- ARM64 device
 - Vulkan 1.0 support
-- Android 7+ (Note: if your device supports Vulkan 1.0, then it is running at least Android 7)
+- Android 7+ (Note: if your device supports Vulkan 1.0, then it is running at least Android 7, and vice versa)
```

Comment:
I'm not sure all the android 7 devices support vulkan

Reply:
fixed, also added some other changes